storage/tianmu/core/data_type.cpp

```diff
@@ -53,6 +53,12 @@ DataType::DataType(common::ColumnType atype, int prec, int scale, DTCollation co
       valtype = ValueType::VT_FIXED;
       fixmax = MAX(TIANMU_TINYINT_MAX, -TIANMU_TINYINT_MIN);
       break;
+    case common::ColumnType::BIT:
+      DEBUG_ASSERT((prec > 0) && (prec <= common::TIANMU_BIT_MAX_PREC));
```

Comment:
Why do we use this `DEBUG_ASSERT` here, and from the code, can draw that we deal with the `bit-value` type as the `bigint` value type.

Reply:
Yeah, this line has been deleted to keep consitent with `bigint`.